ext/symengine/ruby_function.c

```diff
@@ -37,3 +39,39 @@ IMPLEMENT_ONE_ARG_FUNC(gamma);
 
 #undef IMPLEMENT_ONE_ARG_FUNC
 
+VALUE cfunction_functionsymbol_init(VALUE self, VALUE args)
+{
+    int argc = NUM2INT(rb_funcall(args, rb_intern("length"), 0, NULL));
+    printf("argc : %d\n", argc);
+    VALUE first = rb_ary_shift(args);
+    if( TYPE(first) != T_STRING ){
+        rb_raise(rb_eTypeError, "String expected as first argument");
+    }
+    char *name = StringValueCStr( first );
+    char *c;
```

Comment:
@abinashmeher999 all those were used for debugging. I didn't notice that I've added them to the pushed commit. I'll remove them.